src/helpers/profilePictureService.ts

```diff
@@ -1,6 +1,38 @@
 import { PrismaClient } from ".prisma/client";
 import { getEmployeeById } from "./employeeService";
 const prisma = new PrismaClient();
+import dotenv from "dotenv";
+dotenv.config();
```

Comment:
dotenv.config should be called once in your index file

Reply:
Changes for the above comment pushed.